subworkflows/local/prepare_intervals/main.nf

```diff
@@ -26,29 +29,31 @@ workflow PREPARE_INTERVALS {
     intervals_combined   = Channel.empty() // Single bed file containing all intervals
 
     if (no_intervals) {
-        file("${params.outdir}/no_intervals.bed").text        = "no_intervals\n"
-        file("${params.outdir}/no_intervals.bed.gz").text     = "no_intervals\n"
-        file("${params.outdir}/no_intervals.bed.gz.tbi").text = "no_intervals\n"
-
-        intervals_bed        = Channel.fromPath(file("${params.outdir}/no_intervals.bed")).map{ it -> [ it, 0 ] }
-        intervals_bed_gz_tbi = Channel.fromPath(file("${params.outdir}/no_intervals.bed.{gz,gz.tbi}")).collect().map{ it -> [ it, 0 ] }
-        intervals_combined   = Channel.fromPath(file("${params.outdir}/no_intervals.bed")).map{ it -> [ [ id:it.simpleName ], it ] }
-    } else if (params.step != 'annotate' && params.step != 'controlfreec') {
+        file("${outdir}/no_intervals.bed").text        = "no_intervals\n"
+        file("${outdir}/no_intervals.bed.gz").text     = "no_intervals\n"
+        file("${outdir}/no_intervals.bed.gz.tbi").text = "no_intervals\n"
```

Comment:
I am wondering actually if we could dump this into the work dir in the long run.